server/public/plugin/hooks.go

```diff
@@ -278,66 +278,6 @@ type Hooks interface {
 	// Minimum server version: 7.0
 	OnCloudLimitsUpdated(limits *model.ProductLimits)
 
-	// UserHasPermissionToCollection determines if the given user has access to
```

Comment:
This isn't a breaking change, even if you upgrade to the latest `public` module containing this, since the plugin's corresponding implementations will just never get called. But regardless, this was EXPERIMENTAL anyway